docs/pages/guides/using-sentry.md

```diff
@@ -65,12 +67,25 @@ Sentry.init({
   enableInExpoDevelopment: true,
   debug: true, // If `true`, Sentry will try to print out useful debugging information if something goes wrong with sending the event. Set it to `false` in production
 });
+```
+
+#### Usage
+
+Depending on which platform you are (mobile or web), use the following functions in order to capture/catch exceptions:
```

Comment:
```suggestion
Depending on which platform you are on (mobile or web), use the following functions in order to capture/catch exceptions:
```

Reply:
Aside from that, you are not listing functions, so 'the following functions' doesn't really apply. Also, those functions are not _just_ for capturing/catching exceptions, there's methods in there for instrumentation, performance, etc. so a more generic description would be better here.